doc-tool/src/dotty/tools/dottydoc/staticsite/Site.scala

```diff
@@ -311,6 +315,16 @@ case class Site(val root: JFile, val projectTitle: String, val documentation: Ma
     * defaults, the user-defined one will take precedence.
     */
   val layouts: Map[String, Layout] = {
+
+    val cal = Calendar.getInstance()
+    val today = cal.getTime()
+    val currentHour = new SimpleDateFormat("hh").format(today)
+    val currentMinute = new SimpleDateFormat("mm").format(today)
+    val amOrPm = new SimpleDateFormat("a").format(today)
+    val x = "<!-- generated on " + cal.get(Calendar.DATE) + "/"+ (cal.get(Calendar.MONTH)+1)+ "/"+cal.get(Calendar.YEAR) + " " + currentHour + ":" + currentMinute + ":" + amOrPm + "-->"
+    var loc = root.getAbsolutePath.toString + "/.." + "/doc-tool/resources/_layouts/main.html"
+    scala.tools.nsc.io.File(loc).appendAll(x)
```

Comment:
Thanks again and Sorry. Regarding the failing tests, would it be better if I add a check for ```loc ``` over here?